src/integer_mod_q/polynomial_ring_zq/properties.rs

```diff
@@ -0,0 +1,152 @@
+// Copyright © 2023 Marcel Luca Schmidt
```

Comment:
```suggestion
// Copyright © 2024 Marcel Luca Schmidt
```
Minor detail, if you want to update it :)